apps/aot/src/app/events/[year]/[day]/_components/description/index.tsx

```diff
@@ -128,6 +129,8 @@ export function Description({ challenge }: DescriptionProps) {
                   shouldBookmark = true;
                   setHasBookmarked(true);
                 }
+                // TODO: Is this guaranteed to exist, or is userId actually optional?
+                // eslint-disable-next-line @typescript-eslint/no-non-null-asserted-optional-chain
```

Comment:
i need to revisit these types, idk why userId would be optional if `user` is defined